src/bin/CMakeLists.txt

```
@@ -263,7 +265,13 @@ set(PUBLIC_HEADERS
     ../inc/msquic_posix.h
     ../inc/quic_sal_stub.h)
 
-install(TARGETS msquic EXPORT msquic DESTINATION lib)
+if(BUILD_SHARED_LIBS)
+  install(TARGETS msquic EXPORT msquic DESTINATION lib)
+else()
+  install(FILES ${QUIC_STATIC_LIBRARY}
+    DESTINATION lib
+    )
```

This is the real change, right? BTW, one nit:
```suggestion
  install(FILES ${QUIC_STATIC_LIBRARY} DESTINATION lib)
```